src/group/libp2p_group_relcast_server.erl

```diff
@@ -357,7 +357,8 @@ handle_info(force_close, State=#state{}) ->
     %% down the group by exiting the supervisor.
     spawn(fun() ->
                   lager:info("removing group for force_close timeout"),
-                  libp2p_swarm:remove_group(State#state.tid, State#state.group_id)
+                  Mgr = libp2p_group_mgr:remove_group(State#state.tid),
```

Comment:
D you mean `libp2p_group_mgr:mgr` here?

Reply:
lol.  So I reverted the change here totally, and it broke again as expected.  trying with the fix you suggested, I suspect that it will also break.

UPDATE: before I finished typing the above sentence, the "corrected" code caused the same crash.

Reply:
I think it worked because it was failing to close.